db/migrate/20210505195336_remove_requested_documents_later_fields_from_intake.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveRequestedDocumentsLaterFieldsFromIntake < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :intakes, :requested_docs_token, :string
```

Comment:
If possible, it's best to do this in two steps.

1. Fully deploy the new code that doesn't read/write this column
2. Deploy a migration that deletes the column

Doesn't have to be a blocker, but FYI.

Reply:
When deploying, you're going to run into an issue that the `analytics.intakes` table has this column.

The easiest fix IMHO is to adjust `.aptible.yml` so that instead of doing:

```
before_release:
    - bundle exec rake db:migrate
```

it does

```
before_release:
    - bundle exec rake analytics:drop_views db:migrate analytics:create_views
```

I know that [this ticket](https://www.pivotaltracker.com/story/show/177352119) discusses moving the views to Scenic; I don't think that's strictly required, but _something_ is going to have to be done. :)

Reply:
One related idea is you can mark the column as [ignored](https://bigbinary.com/blog/rails-5-adds-active-record-ignored-columns) so that you can confidently delete all the code that references the column.